ui/scss/component/_animation.scss

```diff
@@ -70,11 +70,11 @@
 
 @keyframes menu-animate-in {
   0% {
-    transform: scaleY(0);
+    transform: translateY(-80px);
```

Comment:
This should use `var(--header-height)` (which is 80px)

Reply:
changed, thanks.